django_tasks/backends/database/backend.py

```diff
@@ -43,6 +46,17 @@ def _task_to_db_task(
             backend_name=self.alias,
         )
 
+        # sign value without timestamp
+        # TimestampSigner doesn't support setting our own timestamp?
+        if settings.TASKS.get(self.alias, {}).get("SIGN_TASKS") is True:
+            signer = Signer()
+            _, _, signature = signer.sign_object(db_task_result.canonical).partition(
```

Comment:
Suggestion: In much the same way that user password verification is done, I think signature creation and validation should live on the `TaskResult` - that way the implementation can be shared, and kept in 1 place.

Alternatively, perhaps as separate methods on the backend itself?